workflows/sars-cov-2-variant-calling/sars-cov-2-consensus-from-variation/ro-crate-metadata.json

```diff
@@ -0,0 +1,141 @@
+{
+    "@context": [
+        "https://w3id.org/ro/crate/1.1/context",
+        {
+            "GithubService": "https://w3id.org/ro/terms/test#GithubService",
+            "JenkinsService": "https://w3id.org/ro/terms/test#JenkinsService",
+            "PlanemoEngine": "https://w3id.org/ro/terms/test#PlanemoEngine",
+            "TestDefinition": "https://w3id.org/ro/terms/test#TestDefinition",
+            "TestInstance": "https://w3id.org/ro/terms/test#TestInstance",
+            "TestService": "https://w3id.org/ro/terms/test#TestService",
+            "TestSuite": "https://w3id.org/ro/terms/test#TestSuite",
+            "TravisService": "https://w3id.org/ro/terms/test#TravisService",
+            "definition": "https://w3id.org/ro/terms/test#definition",
+            "engineVersion": "https://w3id.org/ro/terms/test#engineVersion",
+            "instance": "https://w3id.org/ro/terms/test#instance",
+            "resource": "https://w3id.org/ro/terms/test#resource",
+            "runsOn": "https://w3id.org/ro/terms/test#runsOn"
+        }
+    ],
+    "@graph": [
+        {
+            "@id": "./",
+            "@type": "Dataset",
+            "author": "Wolfgang Maier",
+            "datePublished": "2021-06-14T17:44:52.016833",
```

Comment:
Is there any fields that can list the specified version of the workflow ?

Reply:
Yes. A `version` property to be added to the `ComputationalWorkflow` entity. However, what would be the source for the property's value? I see version tags in the `CHANGELOG.md` files, but they're not machine-readable. They're listed in Markdown headings, but some use the atx style (leading hashes) while others use the setext style (underlining with equal signs and dashes). Moreover, some headings only contain the version tag (e.g., `## [0.1]`) while others also contain a date (e.g., `## [0.2] - 2021-04-30`).

We could try parsing the version tags with Python anyway, but that would not be very robust. Alternatives off the top of my head:

* add a `VERSION` file containing only the version string (old style)
* keep the version in the change log, but enforce a strict format for specifying it, e.g., `VERSION <VERSION_TAG>` on the first line or something like that
* keep things as they are now, but at least enforce the atx heading style and write down the convention somewhere as a guide for authors. Currently it seems to be `## [<VERSION>] <OTHER_METADATA>`, so the parsing code should do something like "search for the first level-2 heading, take the first non-whitespace character run and remove the leading and trailing square brackets"

Reply:
There's a release field in the .ga files (which is manually maintained right now, but we're going to allow support for this in Galaxy). This is also what we use the create release tags for deployed repositories on github.

Reply:
Excellent. Workflow version added in f1163a4